### PR TITLE
Desc

### PR DESCRIPTION
**Pull request by disha1202**

### CODE REVIEW

File: src/views/Chatbox.vue [hunk 1]
```diff
@@ -31,20 +31,28 @@
...
+        "
+        placeholder="How do I find peace in the middle of chaos ?"
+        ref="questionInput"
+        v-on:input="validateInput"
+        v-on:focus="validateInput"
+      />
+      <ion-button @click="sendMessage">
+        <ion-icon :icon="paperPlaneOutline"></ion-icon>
+      </ion-button>
+    </form>
```
Comment (disha1202) [inline src/views/Chatbox.vue:55]: Revert unwanted changes

File: src/views/Chatbox.vue [hunk 2]
```diff
@@ -100,6 +108,17 @@ export default {
...
+      let index = 0;
+      const interval = setInterval(() => {
+        if (index < message.length) {
+          this.typingMessage += message[index];
+          index++;
+        } else {
+          clearInterval(interval);
+        }
+      }, 100); // Adjust the typing speed by changing this value (milliseconds)
+    },
```
Comment (disha1202) [inline src/views/Chatbox.vue:121]: Revert unwanted changes

File: src/views/Chatbox.vue [hunk 3]
```diff
@@ -318,8 +336,6 @@ button.button-native {
 
 .messages {
   overflow-y: scroll;
-  height: 70vh;
-  padding-bottom: 30px;
```
Comment (disha1202) [inline src/views/Chatbox.vue:322]: Revert unwanted changes

File: src/views/Chatbox.vue [hunk 4]
```diff
@@ -346,8 +362,6 @@ button.button-native {
   }
   .messages {
     overflow-y: scroll;
-    height: 60vh;
-    padding-bottom: 30px;
```
Comment (disha1202) [inline src/views/Chatbox.vue:350]: Revert this change

File: src/views/HomePage.vue [hunk 5]
```diff
@@ -436,14 +439,18 @@ ul.custom-bullet li::before {
     height: 7vh;
   }
   .footer {
-    font-size: 0.9rem;
-    bottom: 8px;
+    font-size: 0.8rem;
+    bottom: 4px;
+  }
+  .ion-toolbar-header {
+    border: 1px solid red;
```
Comment (disha1202) [inline src/views/HomePage.vue:446]: Remove this border

File: src/views/HomePage.vue [hunk 6]
```diff
@@ -436,14 +439,18 @@ ul.custom-bullet li::before {
...
+    padding: 0px;
   }
 }
 
 @media (max-width: 380px) {
```
Comment (disha1202) [inline src/views/HomePage.vue:451]: Update the max width for this media query

File: src/views/Chatbox.vue [hunk 7]
```diff
@@ -31,20 +31,28 @@
         </transition>
       </div>
       <div class="spinner" v-if="loading">
-        <img style="width: 100px" src="../assets/new.gif" />
+        <img src="../assets/dustbin.png" />
```
Comment (disha1202) [inline src/views/Chatbox.vue:34]: Why have we changed this?

**Comment by disha1202** [comment]
@shivamsherje Remove all the images and revert the changes in package-lock.json

File: src/views/Chatbox.vue (hunk 7 above)
Comment (shivamsherje) [reply src/views/Chatbox.vue:34]: fixed

File: src/views/HomePage.vue (hunk 6 above)
Comment (shivamsherje) [reply src/views/HomePage.vue:451]: done

File: src/views/Chatbox.vue (hunk 1 above)
Comment (shivamsherje) [reply src/views/Chatbox.vue:55]: done

File: src/views/Chatbox.vue (hunk 3 above)
Comment (shivamsherje) [reply src/views/Chatbox.vue:322]: this are removed already in conflicts

File: src/views/Chatbox.vue (hunk 2 above)
Comment (shivamsherje) [reply src/views/Chatbox.vue:121]: ye chiz q hatani he  ?
